apps/docs/content/components/Box/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Box
+description: Use a box to simplify the process of applying one-off styles to an element
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![Three empty cards inside a grey background with rounded corners, which is also offset from the edge and including several annotations. First indicating distance between the background area and canvas edge to indicate margin controls. Second showing padding controls between the cards and background outer edge. Third pointing towards background color and lastly one pointing at the rounded corner to illustrate border radius options.](https://github.com/primer/brand/assets/13340707/211be3c8-0f33-47ba-8236-9b19780d2a86)
+
+## Usage
+
+The box component is a versatile layout component that is intended to simplify the process of adding one-off styles such as padding, margin, background color, and border to (or around) an element.
+
+It provides convenient and comprehensive access to Primer Brand's underlying Primitives, such as the spacing scale and functional design tokens for color and borders.
+
+While it is highly customizable and can be used to create complex layouts, it is important to use it judiciously and only when necessary.
+
+## Options
+
+### Padding
+
+Inner spacing is configurable on all sides or specific directions. Box utilises the CSS padding specification, using logical operators to aid bi-directional reading support.
+
+Additionally, fine-grained control over padding at specific viewports is available to ensure optimal, responsive layout on different screen sizes.
+
+### Margin
+
+Outer spacing is also configurable on all sides or specific directions. Box utilises the CSS margin specification, similarly using logical operators to aid bi-directional reading support.
+
+### Background colors
+
+Box enables first-class customisation of its background-color property. Options are limited to background-oriented, functional design tokens such as `default` and `subtle`, to ensure maximum color contrast with foreground text.
+
+### Border
+
+Box offers various border customisations options, including configuration of `border-style`, `border-radius` and `border-colors`.
+
+In most cases, the available options corrospond to border-oriented design tokens available within the system. The exception is border style, which is limited to a subset of the official CSS `border-style` values.
+
+## Accessibility
+
+Box is a purely decorative element by default. It sets no intrinsic roles, ARIA attributes, or keyboard shortcuts.
+
+Any interactive content within a box should be given their own appropriate accessibility treatment based on the specific workflow of your application.
+
+## Related components
+
+- [Stack](/components/Stack): to enable layout of its immediate children along the vertical or horizontal axis.
```

Comment:
I think we should we add the `Grid` component here too, to list all the layout enablers